sdcm/utils/raft/__init__.py

```diff
@@ -168,6 +186,18 @@ def get_severity_change_filters_scylla_start_failed(timeout: int | None = None)
                                         extra_time_to_expiration=timeout)
         )
 
+    def is_group0_tokenring_num_of_nodes_consistent(self) -> bool:
```

Comment:
does it mean `is_group0_num_of_nodes_consistent_with_token_ring`? Or maybe more generic to serve non-raft case: `is_num_of_nodes_in_token_ring_valid`?